arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -132,9 +132,9 @@ def modelDocumentPullLoader(
                         return None  # allow zipped test case to load normally
 
                 isZipFormat = modelXbrl.fileSource.isZip
-                hasZipFileExtension = modelXbrl.fileSource.type.lower() == ".zip"
+                hasZipFileExtension = modelXbrl.fileSource.type.lower() == ".zip" or (modelXbrl.fileSource.type.lower() == ".xbri" and  disclosureSystemYear >= 2024)
```

Comment:
`.lower()` for `.xbri` seems wrong per [spec](https://www.xbrl.org/Specification/report-package/REC-2023-09-22/report-package-REC-2023-09-22.html#sec-file-extensions).

Reply:
Changed